qa-midrc.planx-pla.net/portal/gitops.json

```
@@ -212,14 +212,35 @@
                 "_structured_reports_count"
               ]
             },
+          "dropdowns": {
+            "download": {
```

```suggestion
            "download_studies": {
```
i think this value has to be the same as one of those `dropdownId` that a button has for this tab